spec/integration/hanami/assets/compiler_spec.rb

```diff
@@ -208,15 +208,6 @@
     expect(target.read).to match %(body {\n  font: 100% Helvetica, sans-serif;\n  color: #fff; }\n)
   end
 
-  it 'uses defined sass cache directory' do
```

Comment:
@cllns Did `sassc` remove their cache or is this an attempt to force this implementation to use our cache?

Reply:
It looks like SassC [initially supported Sass's CacheStore API](https://github.com/sass/sassc-ruby/commit/8994a9a013076aa8e2ce38ddccd3b7c3fe7113fc#diff-eb72405d9cc3ebc91cd8a3be4a292ad1), but that has since been [removed](https://github.com/sass/sassc-ruby/commit/d627aff5cd55e5e0e2263f9dbb29f236bf7345a4). I think the argument for not caching is that `libsass` is fast enough that [caching isn't necessary](https://github.com/sass/libsass/issues/2879#issuecomment-502472321).